components/scream/src/share/io/tests/scorpio_ad_test.cpp

```diff
@@ -0,0 +1,61 @@
+#include <catch2/catch.hpp>
```

Comment:
This test needs to go in `scream/tests`. You cannot use `scream_control` from the `src/share` folder, since it's built _after_ the share folder.